draft-ietf-quic-multipath.md

```diff
@@ -323,13 +323,28 @@ client on a new path, if the server decides to use the new path,
 the server MUST perform path validation ({{Section 8.2 of QUIC-TRANSPORT}})
 unless it has previously validated that address.
 
-If validation succeed, the client can send non-probing, 1-RTT packets
-on the new paths.  In contrast with the specification in
+If validation succeeds, the client can continue to use the path.
+If validation fails, the client MUST NOT use the path and can
+remove any status assiciated to the path initation attempt.
+{{Section 9.1 of QUIC-TRANSPORT}} introduces the concept of
+"probing" and "non-probing" frames. When the  multipath extension
+is negoatiated, the receiption of "non-probing"
+packet on a new path needs to be considered as an path initiation
+attempt that does not impact the path status of any existing
```

Comment:
I am not sure to understand the mention of "path status" of existing paths when initiating a new path.

Reply:
because with migration, you basically open a new path and effectively close the old path at the same path. But we can definitely simply remove the term "path status". Does that help?
```suggestion
attempt that does not impact any existing
```

Reply:
+1 Quentin. Or maybe "does not impact the status of any existing [path]"